test-suite/test-functionality.sh

```diff
@@ -214,6 +214,7 @@ main() {
     then
         local default_tests="
             pconn
+            dead-peer
```

Comment:
This regression test is dedicated to the fixed bug and another logging bug fixed in recent master/v7 commit ea3f56e.